pkg/capabilities/capabilities.go

```diff
@@ -0,0 +1,25 @@
+package capabilities
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/syndtr/gocapability/capability"
+)
+
+func CheckRequiredCapabilities(caps capability.Capabilities, reqCaps []capability.Cap) error {
+	for _, c := range reqCaps {
+		if !caps.Get(capability.EFFECTIVE, c) {
+			return fmt.Errorf("insufficient privileges to run: missing CAP_%s", strings.ToUpper(c.String()))
+		}
+	}
+	return nil
+}
+
+func LoadSelfCapabilities(selfCap capability.Capabilities) (capability.Capabilities, error) {
```

Comment:
I like more the scope of the "original" `getSelfCapabilities` method that inits the struct and loads capabilities.
```suggestion
func Self() (capability.Capabilities, error) {
```